pkg/controllers/scalablenodegroup/v1alpha1/suite_test.go

```diff
@@ -53,24 +54,75 @@ var _ = AfterSuite(func() {
 
 var _ = Describe("Examples", func() {
 	var ns *environment.Namespace
+	var desiredReplicas = ptr.Int32(5)
```

Comment:
IMO we should just configure this explicitly in each test.

Reply:
The reason I am setting this here is, because I want to avoid setting `sng.Spec.Replicas` to default value 0, we might miss some cases where values are not set and we just test the default. May be, it will be better if I rename it to `defaultReplicaCount`?

Reply:
Sounds good!

Reply:
Thought variable rename isn't necessary. I think it's good as is.